index.bs

```diff
@@ -455,6 +464,11 @@ dictionary PictureInPictureEventInit : EventInit {
 The <a>task source</a> for all the tasks queued in this specification is the
 <a>media element event task source</a> of the video element in question.
 
+When an algorithm <dfn lt="queue a picture-in-picture task">queues a
```

Comment:
I wonder if we should just reuse [[queue a media element task]] instead: 
https://html.spec.whatwg.org/multipage/media.html#queue-a-media-element-task

That takes a media element instead it seems... that might make more sense?